### ISSUE TITLE
Ignore the dictionary check after a certain char limit

### ISSUE THREAD

**Issue by danrabbit** [elementary/installer#497]
There's way more info on diceword passwords [here](https://blog.1password.com/toward-better-master-passwords/) but the point is that if your password is over 20 characters long it is probably fine, even if it's made up of dictionary words

**Comment by davidmhewitt** [comment]
Can you give an example of the issue here? With one or sometimes two dictionary words, it complains about the password being based on dictionary words. But as soon as I combine 2-4, it considers the password strong, which makes sense to me.

**Comment by danrabbit** [comment]
Yeah sure. I'm using a passphrase of 4 words over 20 characters and it's complaining about the dictionary check. Something like:

robustreplydictiondescent

**Comment by davidmhewitt** [comment]
And this is for the encryption view in the installer, right?

Typing that exact phrase results in a too short password warning and then a dictionary warning while typing `robustrep`, all of the subsequent characters are accepted without warning, and gradually fill up the strength bar.

**Comment by danrabbit** [comment]
Yes that's correct. Is the difference that you're adding a space here?

**Comment by davidmhewitt** [comment]
I'm not seeing a warning with or without spaces, with many combinations of dictionary words. As soon as I get past around 1.5 words, the dictionary word warning goes away.

The behavior in the create a user dialog in Switchboard is exactly the same too. I've also tried with `en_US` language. So I don't understand what's going on...

**Comment by MrPotatoBobx** [comment]
same here, does it sitll happen to you daneil?
